apps/orams/components/AwardsForm/AwardsForm.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Form } from 'react-redux-form'
+
+import Layout from 'shared/Layout'
+
+import BaseForm from 'shared/form/BaseForm'
+import SubmitForm from 'shared/form/SubmitForm'
+import ErrorBox from 'shared/form/ErrorBox'
+import MultiInput from 'shared/form/MultiInput.js'
+import formProps from 'shared/formPropsSelector'
+import { loadProfile } from 'orams/actions/profileActions'
+
+class AwardsForm extends BaseForm {
+  static propTypes = {
+    action: PropTypes.string,
```

Comment:
missing some propTypes

Reply:
Well are we going to do things 'properly' with style guide or should we just 'quick' merge to show progress? There is a lot of code that can be refactored, but then again it is 57 files :/

Reply:
I agree its probably more important to show progress right now